### PR TITLE
TF-1289-PartII : Handle destroy subscription when logout OIDC

### PR DESCRIPTION
**Pull request by nqhhdev**
TF-1289-PartII : Handle destroy subscription when logout OIDC

### CODE REVIEW

File: lib/features/base/reloadable/reloadable_controller.dart [hunk 1]
```diff
@@ -178,24 +178,14 @@ abstract class ReloadableController extends BaseController {
...
+      } else {
+        _goToLogin(arguments: LoginArguments(LoginFormType.credentialForm));
+      }
     }
   }
 
-  void handleLogoutOidcSuccess(LogoutOidcSuccess success) async {
-    log('ReloadableController::handleLogoutOidcSuccess(): $success');
-    await Future.wait([
-      _deleteAuthorityOidcInteractor.execute(),
```
Comment (dab246) [inline lib/features/base/reloadable/reloadable_controller.dart:188]: only belong to oidc

**Review by sherlockvn — DISMISSED** [review]

**Comment by nqhhdev** [comment]
- Logout OIDC for web

https://user-images.githubusercontent.com/99852347/210704895-d206cfed-e5ea-4401-b2eb-71e35bcc816b.mov

**Comment by nqhhdev** [comment]
- Logout OIDC for mobile (Android)

https://user-images.githubusercontent.com/99852347/210729444-bbb261f6-1573-42ee-8a8e-d081424d24e7.mov

**Review by dab246 — APPROVED** [review]
lgtm

**Pull request merged**